pyfstat/make_sfts.py

```diff
@@ -750,8 +754,7 @@ def run_makefakedata(self):
         and else runs the actual generation code.
         """
```

Comment:
this docstring should go back into `run_makefakedata()`

Reply:
Done in 2f31d5c